pkg/cmd/cli/cmd/login/loginoptions.go

```diff
@@ -99,6 +104,16 @@ func (o *LoginOptions) getClientConfig() (*restclient.Config, error) {
 
 	clientConfig := &restclient.Config{}
 
+	// set request timeout from --request-timeout option
```

Comment:
Is there some way to avoid copying this block of code from `pkg/client/unversioned/clientcmd/client_config.go`?